libzeth/tests/circuits/sha256_test.cpp

```diff
@@ -180,26 +170,16 @@ TEST(TestSHA256, TestHashWithZeroLeg)
         pb, {left, right}, "Block_variable");
 
     libzeth::sha256_ethereum<Field> hasher(
-        pb, ZERO, input_block, result, "Sha256_ethereum");
+        pb, input_block, result, "Sha256_ethereum");
```

Comment:
`Sha256_ethereum` -> `sha256_ethereum` (I know that was like this before, but we don't use caps in annotations so one to fix here)

Reply:
changed the annotation